tests/sqlite/grammar/test_sqlite_select_grammar.py

```diff
@@ -461,3 +461,13 @@ def or_where_null(self):
 
     def select_distinct(self):
         return """SELECT DISTINCT "users"."group" FROM "users\""""
+
+    def test_can_compile_json_where(self):
+        to_sql = self.builder.where("options->age", 18).to_sql()
+        self.assertEqual(to_sql, "SELECT * FROM \"users\" WHERE \"users\".`options`->'$.age' = '18'")
```

Comment:
this syntax is invalid and i cant get this to run on sqlite

Reply:
Weird. I swear I tested it.